### ISSUE TITLE
Support for `ZodTuple` type

### ISSUE THREAD

**Issue by j-murata** [asteasolutions/zod-to-openapi#103]
[`ZodArray`]: https://github.com/colinhacks/zod#arrays
[`ZodTuple`]: https://github.com/colinhacks/zod#tuples
[Supported types]: https://github.com/asteasolutions/zod-to-openapi#supported-types

It would be very helpful if `zod-to-openapi` would support the [`ZodTuple`] type.

Since it does not seem to be included in the [Supported types] at this time, I have no choice but to convert it to [`ZodArray`] in advance with functions like the following:

```js
import { ZodTuple, z } from 'zod';

function zodTupleToArray(schema) {
    if (!(schema instanceof ZodTuple)) {
        throw new TypeError('The passed value is not a `ZodTuple`');
    }
    const { _def: { items, rest }, description } = schema;
    const len = items.length;
    const types = [...items];
    rest && types.push(rest);
    const seed = isSingleType(types) ? types[0] : z.union(types);
    const array = z.array(seed).min(len).describe(description);
    return rest ? array : array.max(len);
}

function isSingleType(items) {
    return items.length === 1 || new Set(items.map(s => s._def.typeName)).size === 1;
}
```

**Comment by AGalabov** [comment]
HI @j-murata  thanks for bringing this up. There shouldn't be a problem to add support for tuples. I'll do my best to do it tomorrow

**Comment by j-murata** [comment]
@AGalabov 
Thank you for your response.
I look forward to the release of the updated version.

**Comment by AGalabov** [comment]
@j-murata it should now be available as part of the [4.5.0 Release](https://github.com/asteasolutions/zod-to-openapi/releases/tag/v4.5.0)
